### PR TITLE
update reviewers list

### PR DESCRIPTION
**Pull request by saikat-royc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Update reviewers
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -5,10 +5,11 @@ approvers:
...
   - leiyiz
 reviewers:
-  - davidz627
   - jingxu97
   - mattcary
   - msau42
   - saad-ali
   - saikat-royc
   - leiyiz
+  - psch
```
Comment (mattcary) [inline OWNERS:14]: Peter's github handle is pwschuurman

File: OWNERS [hunk 2]
```diff
@@ -5,10 +5,11 @@ approvers:
   - saikat-royc
   - leiyiz
 reviewers:
```
Comment (msau42) [inline OWNERS:7]: I think you can also remove saad and jing as they are no longer active in this repo. We can always add them back in the future. It'll help with bot assignments

Comment (saikat-royc) [reply OWNERS:7]: Ack

File: OWNERS (hunk 1 above)
Comment (saikat-royc) [reply OWNERS:11]: ah thanks for catching this

**Pull request merged**